src/components/Form/FormRow.js

```diff
@@ -93,3 +96,17 @@ export const FormRow = ({ children, validation, ...props }) => {
 FormRow.defaultProps = ValidationFormRow.defaultProps;
 
 FormRow.propTypes = ValidationFormRow.propTypes;
+
+export class FormRowOPT extends React.Component {
```

Comment:
I guess you are right, renamed:

`FormRow` -> `HelpFormRow`
`FormRowOPT` -> `FormRow` + added a comment there

`HelpFormRow` is still needed as it will be used in the followup